### ISSUE TITLE
Unable to Edit Pages or Posts - Please Help

### ISSUE THREAD

**Issue by thonyan** [benjaminprojas/wp-editor#38]
I've installed WP-Editor in WordPress 4.5.1 running Twenty-Eleven Child theme.  I'd like to use it to edit the html code on pages.  However, when I open an existing Page or Post, the "Text" tab is inoperable, and there does not appear to be way to edit the code anywhere on the Page or Post code area.

Is there something I'm not getting, or can this be attributed to the version of WordPress to which I recently updated?

Thanks very much for a timely reply (please).

**Comment by euklidis** [comment]
I have the same problem. After installing WP-Editor i cannot add posts with tinyMCE or the default editor of WP. I am running 4.5.2

**Comment by benjaminprojas** [comment]
@thonyan @euklidis 

I am sorry for the delay in responding.  Can you try deactivating all plugins and make sure only the default theme is activated (temporarily) and see if the issues persist?  This will help us determine if there is a plugin or theme conflict.  If the issue goes away, try reactivating all plugins and the theme you are using (one by one) and see if the problem returns.

**Comment by askdesign** [comment]
Hi, 
I noticed a problem today with WP Editor not working when using the Theme Editor, modifying a theme's style.css. WP 4.5.2 is installed. It does work ok on pages and posts. 

I disabled all plug-ins and the problem disappeared. So, adding them back 1 by 1, it was Wordfence that was the culprit. Hope this helps you resolve things.

my best, Anne

**Comment by benjaminprojas** [comment]
Thanks Anne! Are you using WP Editor 1.2.6? It was updated last night. I will try to check the conflict with WordFence.

**Comment by askdesign** [comment]
Yes, WP Editor 1.2.6 was just installed today and that's when the problem arose. On sites where I have the previous version installed (v1.2.5.3), WP Editor still works ok. Glad I was able to help!

**Comment by benjaminprojas** [comment]
@askdesign 

I can't seem to duplicate the error with WordFence and WP Editor.  Can you look at the console when trying to edit the theme file and see if there are Javascript errors?  It could be that I don't have WordFence configured the same way you do so it would be good to try and duplicate what you have.  Can you send me screenshots of your settings (without personal information if applicable)? 

Thanks!

**Comment by askdesign** [comment]
What do you mean look at the console? I can do a screenshot of my settings, if that would be helpful.

**Comment by benjaminprojas** [comment]
Anne,

Yes, screenshots of your WordFence settings would be very helpful.

In the Chrome browser, If you right click on the page where WP Editor is NOT working and click on "inspect":
![image](https://cloud.githubusercontent.com/assets/962530/15295153/300f9110-1b5d-11e6-8a94-bc5740985f80.png)

That should open up the Developer Tools inspector  Then click on the Console tab: 
![image](https://cloud.githubusercontent.com/assets/962530/15295172/4de48330-1b5d-11e6-8856-9099609d49f9.png)

There you should see if there are any errors:  
![image](https://cloud.githubusercontent.com/assets/962530/15295181/5a407dbe-1b5d-11e6-9203-c734c99d60fe.png)

**Comment by theGuruWithin** [comment]
i am experiencing same issue with plugin not working as described above.... I have used plugin forever and love it... can't live without.... I don't run Wordfence but i am running WP 4.5.2

**Comment by askdesign** [comment]
@theGuruWithin My temporary work-around is to disable Wordfence when I am modifying the theme's CSS.

**Comment by theGuruWithin** [comment]
thanks for quick reply
I am not running Wordfence any other suggestions?

**Comment by askdesign** [comment]
You should disable all plug-ins, then activate them 1 by 1, testing WP Editor to see which plug-in causes it to stop working. That's what I did and Wordfence came up as the problem.

**Comment by theGuruWithin** [comment]
i removed every plugin except for WP Editor and problem in Posts and Pages is consistant... WP Editor not working properly. I think it  may be WP 4.5.2 issues since they made some changes to how editor worked in this last update... guess i will just disable till 1.2.7 is released... how sad after all this time of working beautifully... plugin will be missed...

**Comment by benjaminprojas** [comment]
@theGuruWithin I'm sorry to hear about the issues you are having.  The issue may have to do with a conflict with your theme.  Can you try activating one of the default WordPress themes and see if that makes a difference?

**Comment by theGuruWithin** [comment]
Ben,

I love this plugin you really have done an awesome job.

I currently use theme TwentyEleven with Nom Nom child theme addon... (both are up to date)

Since your suggestion I just switched to plain Twenty Eleven with out Nom Nom and also tried switch to Twenty Fifteen theme plain vanilla .... 

Problem is consistent... 

format tag buttons in upper post menu don't work at all... and i see a thin white line on left edge of edit window.  I can send screen grab or any logs etc if you like. 

thanks again for your help
in Gratitude,
Arthur

**Comment by benjaminprojas** [comment]
@theGuruWithin Yes, screenshots would be great, but can you please open a new issue?

Thanks!

**Comment by benjaminprojas** [comment]
@askdesign is this issue still happening for you with the latest release of WP Editor? (1.2.6.2)

**Comment by askdesign** [comment]
Hi Ben,The issue has been resolved with the latest release of WP Editor v1.2.6.2.
Thank you!
Anne /////////////////////////////////////////// 
Anne S. Katzeff 
 Web/Graphic Designer & Artist
 ASK Design 
http://www.askdesign.biz/ 
 /////////////////////////////////////////// 
 Thank you for printing this e-mail only if it is necessary.

Websites are like gardens: they need constant tending.

**Comment by benjaminprojas** [comment]
Thanks Anne!

**Comment by askdesign** [comment]
Oops, sorry, Ben, I spoke too soon. I am still seeing the issue on some sites, not all. Let me see if there's a common thread and I'll get back to you! Apologies!

**Comment by benjaminprojas** [comment]
Okay thanks.

**Comment by jpgklassen** [comment]
I'm having the same issue. When I deactivate WP Editor, everything works again. Sadly I've had to leave it deactivated because of this. :( A fix would be so appreciated as I love using your plugin!!
